src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -947,11 +947,9 @@ Real SpatialPooler::avgConnectedSpanForColumn1D_(UInt column)
   vector<UInt> connectedSparse = connectedSynapses_.getSparseRow(column);
   if (connectedSparse.empty())
     return 0;
-  UInt minIndex = *min_element(connectedSparse.begin(),
+  auto minmax = minmax_element(connectedSparse.begin(),
                                connectedSparse.end());
-  UInt maxIndex = *max_element(connectedSparse.begin(),
-                               connectedSparse.end());
-  return maxIndex - minIndex + 1;
+  return *minmax.second /*max*/ - *minmax.first /*min*/ + 1;
```

Comment:
Thanks for creating the other PR. Can you remove this change from this PR? I want to make sure "git blame" attributes this to the other commit after all of this is merged.